Sources/StreamChatUI/ChatMessageList/ChatThreadVC.swift

```diff
@@ -206,39 +183,44 @@ open class _ChatThreadVC<ExtraData: ExtraDataTypes>:
         layoutOptions.remove(.threadInfo)
         return layoutOptions
     }
+    
+    public var messages: [_ChatMessage<ExtraData>] {
+        messageController.replies + [messageController.message!]
+    }
```

Comment:
By converting the replies here to an array we're effectively losing the lazy loading from core data. Let's deal with it in a follow PR.

Reply:
good catch 👍